docs/gateway-api/gateway-api-v1.md

```diff
@@ -51,7 +51,7 @@ It is important that the `.spec.controllerName` value specified MUST match `ako.
 
 #### Gateway
 
-The Gateway object represents an instance of a service-traffic handling infrastructure by binding Listeners to a set of IP addresses. The AKO validates the Gateway object and updates the status as `Accepted`. Then, the AKO translates the Gateway and its configuration as a Parent VS. The listeners in Gateway will be translated as listeners in the Parent VS and secrets will be configured as Certificates in the AVI controller and will be referenced in the same Parent VS. AKO updates the status of Gateway as `programmed` along with the VIP of the Parent VS once the VS creation is completed.
+The Gateway object represents an instance of a service-traffic handling infrastructure by binding Listeners to a set of IP addresses. The AKO validates the Gateway object and checks if the listeners are valid and sets `ListenerConditionAccepted` to `true`. If one or more listener in a gateway is valid, `GatewayConditionAccepted` is set to `true`. It then checks whether all the references specified in the gateway listeners are valid and exist and then sets `ListenerConditionResolvedRefs` to `true` accordingly. It then translates the Gateway and its configuration to a Parent VS. The listeners in Gateway gets translated to listeners in the Parent VS and secrets will be configured as Certificates in the AVI controller and will be referenced in the same Parent VS. AKO updates the status of Gateway with `GatewayConditionProgrammed` as `true` along with the VIP of the Parent VS once the VS creation is completed and sets `ListenerConditionProgrammed` to `true` if respective listener is programmed.
```

Comment:
`translated to listeners in the Parent VS` --> `is mapped to service ports in Parent VS`

Reply:
Done